src/components/PageElements/PageElements.tsx

```diff
@@ -100,90 +109,125 @@ const PageElements: React.FC<PageElementProps> = ({
+              >
+                {canApplicantEdit && (
+                  <UpdateIcon
+                    onClick={() => push(`/application/${serial}/${sectionCode}/Page${pageNumber}`)}
+                  />
                 )}
-              </Segment>
-              {isChangeRequest && <SummaryReviewResponseElement {...props} />}
-            </div>
+              </ApplicantResponseElement>
+            </RenderElementWrapper>
           )
         })}
       </Form>
     )
   }
 
-  // TODO: Find out problem to display edit button with review responses when Review is locked
-
+  // Review & Consolidation
   if (isReview) {
     return (
-      <Form>
-        {visibleElements.map(
-          ({
-            element,
-            thisReviewLatestResponse,
-            isNewApplicationResponse,
-            latestApplicationResponse,
-          }) => {
-            const props = {
-              isNewApplicationResponse: !!isNewApplicationResponse,
-              latestApplicationResponse: latestApplicationResponse,
-              summaryProps: getSummaryViewProps(element),
-              thisReviewLatestResponse: thisReviewLatestResponse,
-            }
-
-            const isChangeRequest: boolean = !!thisReviewLatestResponse?.decision
+      <div>
+        <Form>
+          {visibleElements.map(
+            ({
+              element,
+              thisReviewLatestResponse,
+              isNewApplicationResponse,
+              isActiveReviewResponse,
+              latestApplicationResponse,
+              latestOriginalReviewResponse,
+            }) => {
+              const toggleDecision = openResponse === element.code
+              const summaryViewProps = getSummaryViewProps(element)
 
-            return (
-              <div key={`${element.code}ReviewContainer`}>
-                <Segment basic key={`question_${element.id}`} className="summary-page-element">
-                  {element.category === TemplateElementCategory.Question ? (
-                    <ReviewResponseElement {...props} />
-                  ) : (
-                    <SummaryInformationElement {...props} />
-                  )}
-                  {thisReviewLatestResponse && (
-                    <ReviewDecisionElement
-                      latestApplicationResponse={latestApplicationResponse}
-                      reviewResponse={thisReviewLatestResponse}
-                      summaryViewProps={props.summaryProps}
-                    />
-                  )}
-                </Segment>
-              </div>
-            )
-          }
-        )}
-      </Form>
+              // Information - no review
+              if (element.category === TemplateElementCategory.Information) {
+                return (
+                  <RenderElementWrapper key={element.code}>
+                    <SummaryInformationElement {...summaryViewProps} />
+                  </RenderElementWrapper>
+                )
+              } else {
```

Comment:
return would exit the method, would be nice to reduce indentation in next block by remove the else